### PR TITLE
Fixes to rendering images

### PR DESCRIPTION
**Pull request by DonJayamanne**
Fixes https://github.com/microsoft/vscode-jupyter/issues/6002#issuecomment-849903016
(converting bytes to base64 is an inefficient approach, using native Blobs)

### CODE REVIEW

File: src/client/index.tsx [hunk 1]
```diff
@@ -51,11 +51,7 @@ function convertVSCodeOutputToExecutResultOrDisplayData(
 ): nbformat.IExecuteResult | nbformat.IDisplayData {
     const isImage = cellInfo.mime.toLowerCase().startsWith('image/');
     const isJson = cellInfo.mime.toLowerCase().includes('json');
-    const value = isImage
-        ? btoa(String.fromCharCode(...new Uint8Array(cellInfo.bytes())))
-        : isJson
-        ? cellInfo.json()
-        : cellInfo.text();
+    const value = isImage ? cellInfo.blob() : isJson ? cellInfo.json() : cellInfo.text();
```
Comment (DonJayamanne) [inline src/client/index.tsx:54]: A more efficient way, using `Blob`

File: src/client/index.tsx [hunk 2]
```diff
@@ -51,11 +51,7 @@ function convertVSCodeOutputToExecutResultOrDisplayData(
 ): nbformat.IExecuteResult | nbformat.IDisplayData {
     const isImage = cellInfo.mime.toLowerCase().startsWith('image/');
     const isJson = cellInfo.mime.toLowerCase().includes('json');
```
Comment (rchiodo) [inline src/client/index.tsx:53]: I thought the isJson function was wrong?

Comment (rchiodo) [reply src/client/index.tsx:53]: Didn't it need to check for actual json? Although I think that's a separate bug.

**Review by rchiodo — APPROVED** [review]

File: src/client/index.tsx (hunk 2 above)
Comment (DonJayamanne) [reply src/client/index.tsx:53]: yes thats wrong, waiting on @IanMatthewHuff not sure if he's working on a fix for that or not.
This issue fixes the image rendering issue.

**Pull request merged**